plutus-core/plutus-core/stdlib/PlutusCore/StdLib/Data/Function.hs

```diff
@@ -79,6 +79,51 @@ applyFun = runQuote $ do
         . lamAbs () x (TyVar () a)
         $ apply () (var () f) (var () x)
 
+{- Note [Recursion combinators]
```

Comment:
Oops, I confused branches and added the Note in this one rather than the `strictify-fix` one. Doesn't matter much, I guess.